allCombined.py

```diff
@@ -91,6 +91,8 @@ def getResult(universityName,url,userName,password):
 			result= s[s.find("Status")+1:s.find(" <img")]
 			result=result[7:]
 		elif(getType==1):
+			result= s[s.find("Status")+1:s.find(" <img")]
+			result=result[7:]
```

Comment:
Why did you add this here? It won't work since it is for the other portal of NEU, see I have used different URL for type 0.

About the types 0 and 1. I thought people check results in 2 phases, 1 whether their documents are in or not and 2 later just their statuses. So, I added that too as a command line para. If you select 0, you will get only information related to application status and if information related to document status if present else, application status only.
Feels kind of odd though. any better alternatives? 

Also, the "Decision Result" in N-You portal is less detailed as it just says whether decision is ready or not, unlike the other url one which tells you what stage it current is in like : just submitted, in review etc.